lib/flapjack/gateways/pager_duty.rb

```diff
@@ -184,26 +185,26 @@ def start
             Kernel.sleep(10)
           end
 
-          # begin
-
-            Zermelo.redis = Flapjack.redis
-
-            loop do
-              @lock.synchronize do
-                # ensure we're the only instance of the PagerDuty acknowledgement check running (with a naive
-                # timeout of five minutes to guard against stale locks caused by crashing code) either in this
-                # process or in other processes
-                if Flapjack.redis.setnx(SEM_PAGERDUTY_ACKS_RUNNING, 'true') == 0
-                  Flapjack.logger.debug("skipping looking for acks in PagerDuty as this is already happening")
-                else
-                  Flapjack.redis.expire(SEM_PAGERDUTY_ACKS_RUNNING, 300)
-                  find_pagerduty_acknowledgements
-                  Flapjack.redis.del(SEM_PAGERDUTY_ACKS_RUNNING)
-                end
+        # begin
+
+          Zermelo.redis = Flapjack.redis
+
+          loop do
+            @lock.synchronize do
+              # ensure we're the only instance of the PagerDuty acknowledgement check running (with a naive
+              # timeout of five minutes to guard against stale locks caused by crashing code) either in this
```

Comment:
@ali-graham ah right, yes. Agreed. Will create an issue, and entertain the idea of implementing it.